docs/community.md

```diff
@@ -79,13 +79,9 @@ Project leads have the following additional privileges:
 * Have "admin" access to the main repository.
 * Can merge any approved PRs reviewed by anyone as they see fit.
 
-Project leads have the following duties, which can also be shared with other team members:
-* Assign reviewers for PRs (if nobody volunteers to be a reviewer), based on the issue label.
-* Manage issue and PR tracker, e.g. close PRs that are abandoned for long.
-* Manage weekly release (i.e. act as Release Lead).
-* Invite new committers/core team members to the relevant teams in GitHub.
-* Ensure no unnecessary branches remain in the main repository.
-* Ensure that there are sufficient beginner-level issues (`d.FirstTimers` and `d.Contributors`) to last for at least 7 days (considering the current level of activity) at any time.
+Project leads have the following duties:
+* Manage weekly release (i.e. act as Release Lead). Can also be delegated to more junior team members.
+* Read and understand [maintainer's duties](maintainer-guide.md), and make sure that they are all covered by the team.
```

Comment:
Is this document meant for all core team members to read or only project leads?

If it's for everyone, then perhaps should mention the `maintainer's duties` section in the senior dev instead.